CRM/Core/Form.php

```diff
@@ -752,6 +752,13 @@ public function getTitle() {
    */
   public function setTitle($title) {
     $this->_title = $title;
+    CRM_Utils_System::setTitle($title);
+    // This assignment is done 'in case someone wants to use it'
+    // which is scary because we are supporting something that
+    // we have no idea if it's used or how to stop it breaking.
+    // Don't add things to the code 'just in case' in future...
+    // For now - let's stdise on it.
+    $this->assign('title', $title);
```

Comment:
@colemanw do you think I could get away with removing it from those forms - it's only Event & Contribution forms & it seems pretty clear to me they load without it

Reply:
I'd be happy with that.

Reply:
Note that if the template really wants this it should be able to access `{$form->_title}`.

Reply:
I've updated it - I *think* the title might have been used in emails but I think that would not be the case anymore since we've made the confirmation emails less form dependent over time

Reply:
(we do have a unit test that should cover contribution pages & the 2 instances I found were online contribution & online membership receipts)

Reply:
OK - I just manually tested a membership with this patch applied & the subject still has the title


MIME-Version: 1.0
Content-Type: multipart/alternative;
 boundary="=_dfb7e738f9c538e22b5821607b7ce95c"
From: Membership Department <memberships@civicrm.org>
To: Eileen McNaughton <demo@example.com>
Subject: [TEST]Receipt - Member Signup and Renewal
Content-Disposition: inline
Content-Transfer-Encoding: 8bit
Return-Path: memberships@civicrm.org
Date: Mon, 11 Mar 2019 21:10:12 +0000